src/codegen/llvm.zig

```diff
@@ -8055,6 +8188,26 @@ fn backendSupportsF80(target: std.Target) bool {
     };
 }
 
+/// This function returns true if we expect LLVM to lower f16 correctly
+/// and false if we expect LLVM to crash if it counters an f16 type or
+/// if it produces miscompilations.
```

Comment:
No, it is best if the user does not check these conditions. Technically, they already can, because they can put this logic into their application code and check for `@import("builtin").zig_backend` being one of the LLVM backends. However, see the doc comment on the `CompilerBackend` enum for best practices related to this.